tools/coqdoc/output.ml

```diff
@@ -661,7 +661,8 @@ module Html = struct
   let reference s r =
     match r with
     | Def (fullid,ty) ->
-        printf "<a name=\"%s\">" (sanitize_name fullid);
+        let s' = sanitize_name fullid in
+        printf "<a name=\"%s\" class=\"idref\" href=\"#%s\">" s' s';
```

Comment:
`name` is apparently obsoleted by `id` now. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

This looks fine otherwise. (Thanks for pointing out this is easy! I didn't realize the other changes were just on tests files)

Reply:
(Little ping.)

```suggestion
        printf "<a id=\"%s\" class=\"idref\" href=\"#%s\">" s' s';
```

And then the tests should be updated.

Reply:
Thanks for the suggestion. I now updated the test files.